dcpy/connectors/edm/publishing.py

```diff
@@ -602,7 +605,12 @@ def _cli_wrapper_upload(
     logger.info(
         f'Uploading {output_path} to {product}/build/{build_name} with ACL "{acl}"'
     )
-    upload(output_path, BuildKey(product, build_name), acl=acl_literal)
+    upload(
+        output_path,
+        BuildKey(product, build_name),
+        acl=acl_literal,
+        max_files=int(max_files),
```

Comment:
does just specifying this as an int in the signature cause any issues?

Reply:
not that I can tell. seems like the logical branches would be
- default value `s3.MAX_FILE_COUNT` which is already an `int`
- a number passed in via CLI like `300` which would I think come in as a `str` so `int("300")`
- a non-number passed in via CLI like `number` which would throw a valid error when doing `int("number")`

Reply:
ah, so what I'm asking is whether we should just let `typer` handle coercion/validation. Which would give us the benefit that the CLI would show the right type in the --help section for the command

Reply:
oops I misunderstood what you meant by "signature". I'll change this to let `typer` handle it